climada/entity/exposures/base.py

```diff
@@ -280,6 +280,46 @@ def check(self):
         except AttributeError:  # no geometry column
             pass
 
+    def get_impf_column(self, haz_type):
+        """Find the best matching column name in the exposures dataframe for a given hazard type,
+
+        Parameters
+        ----------
+        haz_type : str
+            hazard type, as in the hazard's tag.haz_type
+            or the HAZ_TYPE constant of the hazard's module
+
+        Returns
+        -------
+        str
+            a column name, the first of the following that is present in the exposures' dataframe:
+            - impf_[haz_type]
+            - if_[haz_type]
+            - impf_
+            - if_
```

Comment:
These two values are not correct if a `haz_type` has been defined. An empty exposures is initialialized without specific `haz_type`, thus with `if_` or `impf_`. As soon as a `haz_type` is specified, one must use `if_haz_type` (`impf_haz_type`)